### PR TITLE
Postgres aws auth

### PR DESCRIPTION
**Pull request by famarting**
Docs update for this components contrib PR https://github.com/dapr/components-contrib/pull/3597

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

**Comment by famarting** [comment]
I think the DCO has failed not because of my commit but by some other commit already in the v1.15 branch

File: daprdocs/content/en/reference/components-reference/supported-bindings/postgresql.md [hunk 1]
```diff
@@ -65,9 +65,11 @@ The AWS authentication token will be dynamically rotated before it's expiration
...
 | `awsRegion` | Y | The AWS Region where the AWS Relational Database Service is deployed to. | `"us-east-1"` |
-| `awsAccessKey` | Y | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
-| `awsSecretKey` | Y | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
+| `awsAccessKey` | N | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
+| `awsSecretKey` | N | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
 | `awsSessionToken` | N | AWS session token to use. A session token is only required if you are using temporary security credentials. | `"TOKEN"` |
+| `awsIamRoleArn` | N  | IAM role that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```
Comment (JoshVanL) [inline daprdocs/content/en/reference/components-reference/supported-bindings/postgresql.md]:
```suggestion
| `awsIamRoleArn` | N  | IAM role that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```
```suggestion
| `awsIamRoleArn` | N  | IAM role to assume that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```

File: daprdocs/content/en/reference/components-reference/supported-configuration-stores/postgresql-configuration-store.md [hunk 2]
```diff
@@ -91,9 +91,11 @@ The AWS authentication token will be dynamically rotated before it's expiration
...
 | `awsRegion` | Y | The AWS Region where the AWS Relational Database Service is deployed to. | `"us-east-1"` |
-| `awsAccessKey` | Y | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
-| `awsSecretKey` | Y | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
+| `awsAccessKey` | N | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
+| `awsSecretKey` | N | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
 | `awsSessionToken` | N | AWS session token to use. A session token is only required if you are using temporary security credentials. | `"TOKEN"` |
+| `awsIamRoleArn` | N  | IAM role that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```
Comment (JoshVanL) [inline daprdocs/content/en/reference/components-reference/supported-configuration-stores/postgresql-configuration-store.md]:
```suggestion
| `awsIamRoleArn` | N  | IAM role to assume that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```

File: daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md [hunk 3]
```diff
@@ -95,9 +95,11 @@ The AWS authentication token will be dynamically rotated before it's expiration
...
 | `awsRegion` | Y | The AWS Region where the AWS Relational Database Service is deployed to. | `"us-east-1"` |
-| `awsAccessKey` | Y | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
-| `awsSecretKey` | Y | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
+| `awsAccessKey` | N | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
+| `awsSecretKey` | N | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
 | `awsSessionToken` | N | AWS session token to use. A session token is only required if you are using temporary security credentials. | `"TOKEN"` |
+| `awsIamRoleArn` | N  | IAM role that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```
Comment (JoshVanL) [inline daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md]:
```suggestion
| `awsIamRoleArn` | N  | IAM role to assume that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```

File: daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v2.md [hunk 4]
```diff
@@ -95,9 +95,11 @@ The AWS authentication token will be dynamically rotated before it's expiration
...
 | `awsRegion` | Y | The AWS Region where the AWS Relational Database Service is deployed to. | `"us-east-1"` |
-| `awsAccessKey` | Y | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
-| `awsSecretKey` | Y | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
+| `awsAccessKey` | N | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
+| `awsSecretKey` | N | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
 | `awsSessionToken` | N | AWS session token to use. A session token is only required if you are using temporary security credentials. | `"TOKEN"` |
+| `awsIamRoleArn` | N  | IAM role that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```
Comment (JoshVanL) [inline daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v2.md]:
```suggestion
| `awsIamRoleArn` | N  | IAM role to assume that has access to AWS Relational Database Service. This is another option to authenticate with AWS aside from the AWS Credentials. | `"arn:aws:iam::123456789:role/rdsRole"` |
```

**Review by JoshVanL — CHANGES_REQUESTED** [review]

**Review by hhunter-ms — APPROVED** [review]

**Pull request closed without merging**